bucket/nmap.json

```diff
@@ -1,18 +1,26 @@
 {
-    "homepage": "https://nmap.org",
     "version": "7.70",
-    "license": "https://svn.nmap.org/nmap/COPYING",
-    "url": "https://nmap.org/dist/nmap-7.70-setup.exe",
+    "description": "A free and open source utility for network discovery and security auditing.",
+    "homepage": "https://nmap.org",
+    "license": {
+        "identifier": "Freeware",
```

Comment:
GPL-2.0-Only per https://svn.nmap.org/nmap/COPYING

Reply:
It's not only a GPL 2, should I change license to GPL-2.0-only or only chsnge the identifier and keep the url?

Reply:
You can leave the URL:
```
"license": {
"identifier": "GPL-2.0-Only",
"url": ""
}